app/assets/javascripts/common/reports.js

```diff
@@ -1699,3 +1678,21 @@ const intersectDataVerticalLine = {
   },
 };
 
+// ---
+// Object merging functions
```

Comment:
We can axe this comment, I don't think it's required.

Reply:
Removed comment